node-tests/fixtures/ember-test/test-index-extra.js

```diff
@@ -1,18 +1,24 @@
-// Make sure dependencies (node_modules) are loadable
-const fileUrl = require('file-url');
-fileUrl('foo.html');
-// Make sure local libraries are loadable
-const helper = require('../src/helper');
-helper();
+// This gets pasted into the test index file, so put our code inside a function
+// to make sure we don't have naming conflicts with already-declared variables
+function testIndexExtra() {
+  // Make sure dependencies (node_modules) are loadable
+  require('electron-is-dev');
 
-const { app, BrowserWindow } = require('electron');
+  // Make sure local libraries are loadable
+  const helper = require('../src/helper');
+  helper();
 
-app.once('browser-window-created', (event, win) => {
-  win.webContents.on('did-finish-load', () => {
-    let extensions = BrowserWindow.getDevToolsExtensions();
-    if (!Object.prototype.hasOwnProperty.call(extensions, 'devtron') || !Object.prototype.hasOwnProperty.call(extensions, 'Ember Inspector')) {
-      console.error('devtron and/or ember-inspector not installed', Object.keys(extensions)); // eslint-disable-line no-console
-      app.exit(-1);
-    }
+  const { app, BrowserWindow } = require('electron');
+
+  app.once('browser-window-created', (event, win) => {
+    win.webContents.on('did-finish-load', () => {
+      let extensions = BrowserWindow.getDevToolsExtensions();
+      if (!Object.prototype.hasOwnProperty.call(extensions, 'devtron') || !Object.prototype.hasOwnProperty.call(extensions, 'Ember Inspector')) {
+        console.error('devtron and/or ember-inspector not installed', Object.keys(extensions)); // eslint-disable-line no-console
+        app.exit(-1);
+      }
+    });
   });
-});
+}
+
+testIndexExtra();
```

Comment:
Would we ever export or reuse this function? I'm wondering if there's a reason (other than stylistic preference, which is a reason I'm OK with in your case :D) not to use an IIFE instead.

Reply:
Just stylistic -- I've always found IIFEs a bit hard to read, so generally only use them if I need them to generate a value in an object literal or something...no strong preference, though.